README.md

```diff
@@ -249,74 +335,148 @@ podTemplate(yaml: """\
       - name: busybox
         image: busybox
         command:
-        - cat
-        tty: true
-    """.stripIndent()) {
+        - sleep
+        args:
+        - 99d
```

Comment:
Ditto, though it seems

```bash
docker run --rm busybox sleep 99d
```

does work in this case. (I had thought Busybox was one of the cases where it did not. Not sure any more.)

Reply:
I think there were some differences wrt handling `sleep infinity` but I couldn't find any non-working example with `99d`